lib/cook_book.rb

```diff
@@ -0,0 +1,31 @@
+class CookBook
+  attr_reader :recipes
+
+  def initialize
+    @recipes = []
+  end
+
+  def add_recipe(recipe)
+    @recipes << recipe
+  end
+
+  def ingredients
+    list = []
+    @recipes.each do |recipe|
+      recipe.ingredients.each do |ingredient|
+        list << ingredient.name
+      end
+    end
+    list.uniq
+  end
+
+  def highest_calorie_meal
+    @recipes.max_by do |recipe|
+      recipe.total_calories
+    end
+  end
+
+  def date
+    Time.now.strftime("%m-%d-%Y")
```

Comment:
So close!!!! Great job using the strftime method. However, with this as a method, any day I call the method, it will spit out that day, not the day the cookbook was created. Instead, this should be an instance variable in the initialize.